tutorials/hyperspectral/Plot-Hyperspectral-Spectra.Rmd

```diff
@@ -113,15 +100,17 @@ wavelengths <- wavelengths*1000
 
 Next, we will extract all reflectance values for one pixel. This makes up the 
 spectral signature or profile of the pixel. To do that, we'll use the `h5read` 
-function.
+function. Here we pick an arbitrary pixel at `(100,35)`, and use the `NULL` 
+value to select _all_ bands from that location.
```

Comment:
Use *txt* instead of _txt_ for italics, please.

Reply:
Right! Changed here.